corehq/apps/reports/tasks.py

```diff
@@ -337,11 +338,33 @@ def extract_form_info(form, properties=None, case_ids=case_ids):
 
         return form_info
 
-    key = [domain, app_id, xmlns]
-    form_ids = {f['id'] for f in XFormInstance.get_db().view("attachments/attachments",
-                                                             start_key=key + [startdate],
-                                                             end_key=key + [enddate, {}],
-                                                             reduce=False)}
+    if not export_is_legacy:
+        query = (FormES()
+                 .domain(domain)
+                 .app(app_id)
+                 .xmlns(xmlns)
+                 .submitted(gte=parse(startdate), lte=parse(enddate))
+                 .remove_default_filter("has_user"))
+        form_ids = set()
+        for form in query.scroll():
+            try:
+                for attachment in form['_attachments'].values():
```

Comment:
@millerdev Do you have a suggested workflow for keeping our branches in sync? Might just be easy enough to keep track of mentally. Otherwise we could create some sort of conflict resolution like branch, but someone would still need to remember to merge that in to the appropriate branch when the time comes. It also seems likely that this branch will be merged before yours, so maybe we could just go ahead and cherry pick the relevant commit into your branch now. Thoughts?